stdlib/ysh/args.ysh

```diff
@@ -195,13 +206,32 @@ func parseArgs(spec, argv) {
             if (_status !== 0) {
               error "Expected Float after '$arg', got '$[argv[i]]'" (code=2)
             }
+          } elif (flag.type is List[Float]) {
+            setvar i += 1
+            if (i >= len(argv)) {
+              error "Expected Float after '$arg'" (code=2)
+            }
+
+            setvar value = args[flag.name] if (flag.name in args) else []
+            try { call value->append(float(argv[i])) }
```

Comment:
oh actually this reminds me that I want to make an accum() method on dicts:

```
call mydict->accum(flag.name, float(argv[i]))
```

which will do the empty list logic

I find that pattern to be quite common.  I think Perl  might have something like this, but Python/JS don't

And it is a trivial method ... not sure if accum() is the best name

----

Although maybe this was summing an integer, not appending to a list ... I'll start a thread on #language-design about it